README.md

```diff
@@ -174,6 +174,26 @@ Get the `SpanID` of the `activeSpan` in the `Context`.
 go2sky.SpanID(ctx)
 ```
 
+## Application Log  Report
+
+```go
+
+r, err := reporter.NewGRPCReporter("oap-skywalking:11800")
+if err != nil {
+    log.Fatalf("new reporter error %v \n", err)
+}
+defer r.Close()
+
+xSkyapmError,xSkyapmLogger:=go2sky.NewSkyLogger(r)
```

Comment:
The `logger` should be the first returen value, the err should be 2nd, I think. According to golang style

Reply:
fixed